client/src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,42 @@
+import { Link, useResolvedPath } from "react-router-dom";
+import SearchBox from "../tools/SearchBox";
+import FooterNav from "../nav/FooterNav";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFacebook, faTwitter, faInstagram, faTiktok } from "@fortawesome/free-brands-svg-icons";
+import HomeLogo from "../graphics/HomeLogo";
+
+type Props = {
+  topics: string[];
+};
+
+const Footer = ({ topics }: Props) => {
+
+  return (
+    <div className="flex justify-center align-center bg-black mt-10">
+      <div className="w-[80em]">
+        <div className="py-8">
+          <SearchBox />
+        </div>
+          <FooterNav topics={topics}/>
+        <div className="mt-5 flex">
+          <HomeLogo size={3}/>
+        <div className="text-white ml-auto flex items-center">
+          <FontAwesomeIcon className="mr-5" icon={faFacebook} size={"xl"} />
+          <FontAwesomeIcon className="mr-5" icon={faTwitter} size={"xl"} />
+          <FontAwesomeIcon className="mr-5" icon={faInstagram} size={"xl"} />
+          <FontAwesomeIcon icon={faTiktok} size={"xl"} />
+          </div>
+        </div>
+        <hr className="h-px my-8 bg-gray-600 border-0 dark:bg-gray-700"></hr>
+        <p className="text-slate-400 text-sm mb-5">
+          © 2023 Artificial Intelligence Daily News. Produced by AI-News-Team.
```

Comment:
Can we compute the date, rather than hard code 2023?

```ts
const year = new Date().getFullYear();
```

```tsx
<p>© {year} Artificial Intelligence...</p>
```

Reply:
Nice, I was having trouble with the footer sticking, your solution worked well, cheers.  I have adjusted the date as suggested.

I also noticed in your image the logos in the nav and footer were actually displaying at the wrong size, I found the same happened for me when I stopped and started the front end, with tailwind not picking up a variable within a className string.  I have found a solution but it's a bit messy, hopefully will do for now.